### PR TITLE
Updating rmb-ma's work

### PR DESCRIPTION
**Pull request by Marc-AntoineA**
* Updates from Ilmenau
* Integration of robotic arm and `trashcan_emptying_behavior`
* Creation of a absolute coordinates fake detector
* Visualization tools

### CODE REVIEW

File: baker_brush_cleaning_module_interface/PackageList.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...eidon/Poseidon.package false false;RangeFinderSimulator /opt/MIRA-commercial/domains/sensors/RangeFinderSimulator/RangeFinderSimulator.package false false;FlowControl /opt/MIRA-commercial/domains/application/FlowControl/FlowControl.package false false;SCITOSSimulator /opt/MIRA-commercial/domains/robot/SCITOSSimulator/SCITOSSimulator.package false false;SCITOS /opt/MIRA-commercial/domains/robot/SCITOS/SCITOS.package false false;SCITOSConfigs /opt/MIRA-commercial/domains/robot/SCITOSConfigs/SCITOSConfigs.package false false;WaypointVisitor /opt/MIRA-commercial/domains/tutorials/WaypointVisitor/WaypointVisitor.package false false;BakeRConfig /home/rmb/git/care-o-bot-kinetic/src/baker_intern/BakeRConfig/BakeRConfig.package false false;PilotWallDistance /home/rmb/opt/BakerTest/domains/navigation/PilotWallDistance/PilotWallDistance.package false false;PilotSmoothTransition /home/rmb/opt/BakerTest/domains/navigation/PilotSmoothTransition/PilotSmoothTransition.package false false;BakeRConfig /home/rmb/opt/BakerTest/domains/robot/BakeRConfig/BakeRConfig.package false false;BakeRCleaningModule /home/rmb/opt/BakerTest/domains/actors/BakeRCleaningModule/BakeRCleaningModule.package false false;
```
Comment (ipa-rmb) [inline baker_brush_cleaning_module_interface/PackageList.txt]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_vacuum_cleaning_module_interface/PackageList.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...or.package false false;FlowControl /opt/MIRA-commercial/domains/application/FlowControl/FlowControl.package false false;SCITOSSimulator /opt/MIRA-commercial/domains/robot/SCITOSSimulator/SCITOSSimulator.package false false;SCITOS /opt/MIRA-commercial/domains/robot/SCITOS/SCITOS.package false false;SCITOSConfigs /opt/MIRA-commercial/domains/robot/SCITOSConfigs/SCITOSConfigs.package false false;WaypointVisitor /opt/MIRA-commercial/domains/tutorials/WaypointVisitor/WaypointVisitor.package false false;BakeRConfig /home/rmb/git/care-o-bot-kinetic/src/baker_intern/BakeRConfig/BakeRConfig.package false false;PilotWallDistance /home/rmb/opt/BakerTest/domains/navigation/PilotWallDistance/PilotWallDistance.package false false;PilotSmoothTransition /home/rmb/opt/BakerTest/domains/navigation/PilotSmoothTransition/PilotSmoothTransition.package false false;BakeRConfig /home/rmb/opt/BakerTest/domains/robot/BakeRConfig/BakeRConfig.package false false;BakeRCleaningModule /home/rmb/opt/BakerTest/domains/actors/BakeRCleaningModule/BakeRCleaningModule.package false false;BakeRVacuumCleaningModule /home/rmb/opt/BakerTest/domains/actors/VacuumCleaningModule/BakeRVacuumCleaningModule.package false false;
```
Comment (ipa-rmb) [inline baker_vacuum_cleaning_module_interface/PackageList.txt]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_wet_cleaning_application/resources_lab_ipa/logs/log_2019_23_3_run1.json [hunk 3]
```diff
@@ -0,0 +1,167 @@
+{
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/resources_lab_ipa/logs/log_2019_23_3_run1.json]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_wet_cleaning_application/resources_lab_ipa/logs/log_2019_26_3_run1.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/resources_lab_ipa/logs/log_2019_26_3_run1.json]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_wet_cleaning_application/resources_lab_ipa/logs/log_2019_31_4_run1.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/resources_lab_ipa/logs/log_2019_31_4_run1.json]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_wet_cleaning_application/resources_test/logs/log_2018_29_3_run1.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/resources_test/logs/log_2018_29_3_run1.json]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_wet_cleaning_application/resources_test/logs/log_2019_20_3_run1.json [hunk 7]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/resources_test/logs/log_2019_20_3_run1.json]: please remove this file from the pull request if there was no specific reason why we would need it.

File: baker_wet_cleaning_application/scripts/dirt_removing_behavior.py [hunk 8]
```diff
@@ -26,20 +28,21 @@ def __init__(self, behavior_name, interrupt_var, move_base_service_str, map_acce
...
+		self.use_vacuum_cleaner_ = False
 
 
 	# Method for setting parameters for the behavior
 	def setParameters(self, dirt_position):
 		self.dirt_position_ = dirt_position
 
 
-	# Method for returning to the standard pose of the robot
+	# Method for retcomputeBestAccessiblePoseurning to the standard pose of the robot
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/scripts/dirt_removing_behavior.py]: remove typo

File: baker_wet_cleaning_application/scripts/readme.txt [hunk 9]
```diff
@@ -1,177 +0,0 @@
-================================================================================================
```
Comment (ipa-rmb) [inline baker_wet_cleaning_application/scripts/readme.txt:1]: Why was the readme removed? I would leave it.

File: mira-simulation-config/etc - bak/EstarPlanner.xml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+<root>
```
Comment (ipa-rmb) [inline mira-simulation-config/etc - bak/EstarPlanner.xml]: the whole folder etc - bak should not be on github

**Review by ipa-rmb — CHANGES_REQUESTED** [review]
Hi Marc-Antoine, I just checked the pull request and found a few little things. Could you still change it or should I send you a pull request with my preferred changes?

Please see my comments for change request.

Additionally, some further files are tracked unnecessarily:
- ```baker_wet_cleaning_application/resources_test/maps/map_4.png```
- all files in ```mira-simulation-config/etc - bak``` should not be tracked, excpet they are really the latest versions for backup reasons, then the folder should be renamed to etc-bak
- ```mira-simulation-config/maps/lab_ipa/lab_ipa (copy).png ```
- ```mira-simulation-config/maps/lab_ipa/lab_ipa_clean (copy).png```

File: baker_wet_cleaning_application/scripts/readme.txt (hunk 9 above)
Comment (Marc-AntoineA) [reply baker_wet_cleaning_application/scripts/readme.txt:1]: Fixed. I updated the readme and renamed it `readme.md` to support markdown.

**Comment by Marc-AntoineA** [comment]
Everything should be fine now. I removed the unnecessary tracked files.

**Comment by Marc-AntoineA** [comment]
@ipa-rmb This PR (and all the relative work) will be deleted at the end of February.

**Review by ipa-rmb — APPROVED** [review]
Many thanks for your thorough and comprehensive work!

**Pull request merged**